### PR TITLE
[DOCS] Added more clear english sentence for task api detailed param

### PR DESCRIPTION
**Pull request by amitmbm**
Fix for #88469

### CODE REVIEW

**Comment by amitmbm** [comment]
@DaveCTurner Please review, Thanks in advance :)

File: docs/reference/cluster/tasks.asciidoc [hunk 1]
```diff
@@ -141,8 +141,8 @@ If the parent isn't found, the API does not return a 404.
 ===== Get more information about tasks
 
 You can also use the `detailed` request parameter to get more information about
-the running tasks. This is useful for telling one task from another but is more
-costly to execute. For example, fetching all searches using the `detailed`
+the running tasks. This is useful for distinguish one task from another task but
+is more costly to execute. For example, fetching all searches using the `detailed`
```
Comment (DaveCTurner) [inline docs/reference/cluster/tasks.asciidoc:145]: How about this?

```suggestion
the running tasks. This is useful to distinguish tasks from each other but
is more costly to execute. For example, fetching all searches using the `detailed`
```

Comment (amitmbm) [reply docs/reference/cluster/tasks.asciidoc]: Looks better thanks @DaveCTurner Committed suggested change.

**Review by pxsalehi — APPROVED** [review]
LGTM!

**Comment by pxsalehi** [comment]
The revert message referenced above was an accident. Nothing was reverted!

**Comment by DaveCTurner** [comment]
Thanks @amitmbm & @pxsalehi!

**Pull request merged**